pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "admin_backend"
-version = "2.17.16"
+version = "2.17.17"
```

Comment:
This is out of date now :(

Reply:
Not that that matters actually, because auto tagging only happens when you merge into main, and this is merging back into a branch that doesn't match that pattern 💡